### ISSUE TITLE
is EgressRule still working ?

### ISSUE THREAD

**Issue by pmerlin** [IBM/istio101#149]
pmerlin$ kubectl apply -f analyzer-egress.yaml
error: unable to recognize "analyzer-egress.yaml": no matches for config.istio.io/, Kind=EgressRule

i'm using Istio 1.5.

not able to speak with gateway.watsonplatform.net

it looks like CRD have changed for APi version
I tried several  v1alpha3 . 

Looking at the doc EgressRule seems to have been replace by Destination Rules

**Comment by pmerlin** [comment]
your telling 
In exercise 3, we set up some egress rules to allow the guestbook service to call the Watson Tone Analyzer service created in Exercise 3. By default Istio blocks calls to services outside the service mesh. In order for calls to reach the Watson service, we applied the VirtualService and ServiceEntry found in /istio101/workshop/plans/analyzer-egress.yaml.

it looks like the file /istio101/workshop/plans/analyzer-egress.yaml. is an old version ....
